src/components/primitive/button/button.component.tsx

```diff
@@ -31,7 +31,7 @@ export const Button: React.FunctionComponent<ButtonComponentProps> = (props: But
    * custom styling goes here.
    */
   const StyledButtonComponent = styled(StyledButton)<ButtonComponentProps>`
-    background-color: ${({ color, theme: { colors } }) => color && colors[color]} !important;
+    background: ${({ color, theme: { colors } }) => color && colors[color]} !important;
```

Comment:
What happens if colors[color] is undefined or if color is undefined? Will it print `background: undefined !important;` ?

Reply:
That's the option props. all the custom styling goes here. if the color is undefined, the Primary color will be rendered. i.e coming from button.component.styles.ts file.